code/modules/shuttle/emergency.dm

```diff
@@ -91,13 +91,26 @@
 	var/datum/announcement/priority/emergency_shuttle_called = new(0, new_sound = sound('sound/AI/shuttlecalled.ogg'))
 	var/datum/announcement/priority/emergency_shuttle_recalled = new(0, new_sound = sound('sound/AI/shuttlerecalled.ogg'))
 
+	var/canRecall = TRUE//no bad condom, do not recall the crew transfer shuttle!
+
+
 /obj/docking_port/mobile/emergency/register()
 	if(!..())
 		return 0 //shuttle master not initialized
 
 	shuttle_master.emergency = src
 	return 1
 
+/obj/docking_port/mobile/emergency/Destroy(force)
+	if(force)
+		// This'll make the shuttle subsystem use the backup shuttle.
+		if(src == shuttle_master.emergency)
```

Comment:
Change this conditional to `if(shuttle_master.emergency == src)` please. It's better to have the comparison operator on the right side of the variable being compared to the current instance.